### PR TITLE
Add Store package

### PR DESCRIPTION
**Pull request by gmkv**
This change introduces a bookkeeping application for a store or a warehouse.
Using the `store` package, you may maintain a list of wares in stock, track their quantity,
record orders and produce simple invoices.

### CODE REVIEW

File: store-app/store/store_test.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		t.Errorf(err.Error())
+	}
+
+	order, err := wh.Sell("Apples", 3)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+	// looks like order doesn't need to have the whole Item
+	// must edit it
+	expectedOrder := &store.Order{
```
Comment (mgenov) [inline store-app/store/store_test.go]: want :=

File: store-app/store/store_test.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+	_, err := wh.Sell("Apples", 3)
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+
+	item := wh.StockList["Apples"]
+	expectedRemaining := 2
+	if item.CurrentQuantity != expectedRemaining {
+		t.Errorf("Store doesn't correctly update quantity after successful sale:\ngot=%v\nexpected=%v",
```
Comment (mgenov) [inline store-app/store/store_test.go]: It's not required as the name of the func indicates which test was succeeded or failed.

In go is mostly used got/want notation for assertion.

https://golang.org/doc/code.html#Testing

image

File: store-app/store/store_test.go [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+		t.Errorf("Unexpected error from Sell: %v", err)
+	}
+
+	revenue, err := wh.RevenueFrom("Apples")
+	if err != nil {
+		t.Errorf("Unexpected error from RevenueFrom: %v", err)
+	}
+
+	expectedRevenue := 0.90
+	if revenue != expectedRevenue {
```
Comment (mgenov) [inline store-app/store/store_test.go]: You can shorten it a bit with:
```golang
if want := 0.90; revenue != want {

}
```

**Comment by gmkv** [comment]
I have updated the unit tests.

**Pull request merged**